app/javascript/src/LoginPage/utils/formValidation.js

```diff
@@ -39,20 +41,22 @@ const constraints = {
   }
 }
 
-const validateAllFields = (form) => {
+const validateAllFields = (form: HTMLFormElement) => {
   const errors = validate(form, constraints[form.id])
```

Comment:
What does validate returns if not Object?

Reply:
Returns `undefined` when no errors found